### PR TITLE
One var fit model

### PR DESCRIPTION
**Pull request by lilyclements**

### CODE REVIEW

**Comment by lilyclements** [comment]
@africanmathsinitiative/instat this is ready for review
Side note - I am going to continue work on this, however, I can not until clsucrdistributions pull request is merged

File: instat/dlgOneVarFitModel.vb [hunk 1]
```diff
@@ -196,8 +209,47 @@ Public Class dlgOneVarFitModel
         clsRTTest.AddParameter("conf.level", nudCI.Value.ToString)
     End Sub
 
+    Private Sub SetVarTest()
+        clsVarTest.SetRCommand("EnvStats::varTest")
```
Comment (dannyparsons) [inline instat/dlgOneVarFitModel.vb]: Is this a package we need to include?

File: instat/dlgOneVarFitModel.vb [hunk 2]
```diff
@@ -196,8 +209,47 @@ Public Class dlgOneVarFitModel
...
+        'clsRConvert.SetRCommand("as.vector")
+        'clsRConvert.AddParameter("x", clsRFunctionParameter:=UcrReceiver.GetVariables())
+        'clsRWilcoxTest.AddParameter("x", clsRFunctionParameter:=clsRConvert)
+        clsRWilcoxTest.AddParameter("x", clsRFunctionParameter:=UcrReceiver.GetVariables())
+        clsRWilcoxTest.AddParameter("conf.level", nudCI.Value.ToString)
+        clsRWilcoxTest.AddParameter("mu", nudHyp.Value.ToString)
+    End Sub
+
+    Private Sub SetNonSignTest()
+        clsRNonSignTest.SetRCommand("signmedian.test:: signmedian.test")
```
Comment (dannyparsons) [inline instat/dlgOneVarFitModel.vb]: and this package?

File: instat/dlgOneVarFitModel.vb [hunk 3]
```diff
@@ -425,8 +501,13 @@ Public Class dlgOneVarFitModel
         SetBinomialTest()
     End Sub
 
-    Private Sub cboVariables_TextChanged(sender As Object, e As EventArgs) Handles cboVariables.TextChanged
+    Private Sub cboVariables_TextChanged() Handles cboVariables.NameChanged
```
Comment (dannyparsons) [inline instat/dlgOneVarFitModel.vb]: Is cboVariables a user control? If so, it should have ucr as it's prefix.

File: instat/dlgOneVarFitModel.vb (hunk 1 above)
Comment (lilyclements) [inline instat/dlgOneVarFitModel.vb]: So, Roger wants to have varTest and signmedian.test as hypothesis tests and these are the only packages that contain these (I think! They're small packages?)
Note that varTest is different to var.test (F test)

File: instat/dlgOneVarFitModel.vb (hunk 3 above)
Comment (lilyclements) [inline instat/dlgOneVarFitModel.vb]: Yes it is, I'll change that!

File: instat/dlgOneVarFitModel.vb (hunk 1 above)
Comment (dannyparsons) [inline instat/dlgOneVarFitModel.vb]: No problem. Just need to know which new packages need adding.

**Pull request merged**